code/modules/clothing/suits/armor.dm

```diff
@@ -441,3 +441,21 @@
 	src.icon_state = "reactiveoff"
 	src.item_state = "reactiveoff"
 	..()
+
+/obj/item/clothing/suit/armor/crusader
+	name = "crusader armor"
+	desc = "God will protect those who belive."
+	icon_state = "crusader_suit"
+	item_state = "crusader_suit"
+	slowdown = 0.3
+	matter = list(MATERIAL_BIOMATTER = 25, MATERIAL_PLASTEEL = 10, MATERIAL_STEEL = 15, MATERIAL_GOLD = 2)
+	armor = list(
+		melee = 70,
+		bullet = 50,
```

Comment:
People have been complaining about how powerful crusader armor is. Since this is intended to be used in melee battles, let's decrease it's bullet armor down to standard vest levels.
```suggestion
		bullet = 30,
```

Reply:
Done.